dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -8925,10 +8975,11 @@ CONFIG_SECURITY_YAMA=y
 CONFIG_SECURITY_SAFESETID=y
 CONFIG_SECURITY_LOCKDOWN_LSM=y
 CONFIG_SECURITY_LOCKDOWN_LSM_EARLY=y
-CONFIG_LOCK_DOWN_KERNEL_FORCE_NONE=y
+# CONFIG_LOCK_DOWN_KERNEL_FORCE_NONE is not set
 # CONFIG_LOCK_DOWN_KERNEL_FORCE_INTEGRITY is not set
-# CONFIG_LOCK_DOWN_KERNEL_FORCE_CONFIDENTIALITY is not set
+CONFIG_LOCK_DOWN_KERNEL_FORCE_CONFIDENTIALITY=y
```

Comment:
IIRC there were some problems with lockdown, it locked down too much for us (e.g. debugfs, maybe some other things we use). What exactly does CONFIDENTIALITY level disable?